src/views/Welcome/Welcome.tsx

```diff
@@ -1,5 +1,64 @@
-import React from 'react';
+import React, {useCallback} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {useGame} from '@contexts/game.context';
+import {MAX_PLAYERS, PlayerColorBank} from '@components/app/consts';
+import openings from '@assets/openings.json';
+
+const cetegories = Object.keys(openings);
 
 export const Welcome = (): React.JSX.Element => {
-    return <h1>Welcome</h1>;
+    const navigate = useNavigate();
+    const { config: { players, openerCategory }, setConfig } = useGame();
+    const inputRef = React.useRef<HTMLInputElement>(null);
+
+    const onCategoryChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+        setConfig((prevState) => ({...prevState, openerCategory: event.target.value as Game['openerCategory']}));
+    }, [setConfig]);
+
+    const onPlayClick = useCallback(() => {
+        const isValid = players.every((player) => player.name.trim().length);
+        if (isValid) {
+            navigate('/');
```

Comment:
I think the welcome page should be rerouted to from empty and the game should be assigned another route , what do you think ?